src/initializers/swagger.initializer.ts

```diff
@@ -18,21 +19,38 @@ export interface SwaggerOptions {
     type?: string | Constructor | Constructor[],
 }
 
+export function normalizeArraySizeOptions<Options extends SwaggerOptions>(
+    options: Options,
+): ArraySizeOptions | undefined {
+    if (!options.isArray) {
+        return undefined;
+    }
+
+    if (options.isArray === true) {
+        return {};
+    }
+
+    return options.isArray;
+}
+
 export function initializeSwagger<Options extends SwaggerOptions>(
     options: Options,
 ): PropertyDecorator[] {
+    const isArray = normalizeArraySizeOptions(options);
+
     return [
         ApiProperty({
             enum: options.enum,
             enumName: options.enumName,
             description: options.description,
             format: options.format,
+            isArray: !!options.isArray,
```

Comment:
It turns out that `@nestjs/swagger` supports both `isArray` and `type: [Foo]`; we're now switching to the former for simplicity.

The resulting OpenAPI spec mostly works. I see that when I set `format: 'uuid'` and `isArray: true`, I get something like:

```js
{
     "format": "uuid",
     "items": {
         "type": "string",
      },
      "type": "array",
}
```

instead of:

```js
{
     "items": {
         "format": "uuid",
         "type": "string",
      },
      "type": "array",
}
```

This seems like a bug in `@nestjs/openapi` and not something we can fix at this layer.

Reply:
This was supposedly fixed: https://github.com/nestjs/swagger/issues/172

Reply:
Now filed again as: https://github.com/nestjs/swagger/issues/1178